caput/coordinates/sphcoords.py

```diff
@@ -1,4 +1,8 @@
-"""Useful functions for radio interferometry.
+"""Useful functions for dealing with spherical coordinates.
```

Comment:
Looking through the `ch_pipeline` PR, I think this should be renamed `caput.coordinates.spherical`.  You don't need the `coords` in the name here; that's already provided by the parent module.